torchvision/prototype/models/depth/stereo/crestereo.py

```diff
@@ -1055,11 +1057,145 @@ def _crestereo(
     return model
 
 
+_COMMON_META = {
+    "min_size": (384, 512),
+}
+
+
 class CREStereo_Base_Weights(WeightsEnum):
-    pass
+    """The metrics reported here are as follows.
+
+    ``mae`` is the "mean-average-error" and indicates how far (in pixels) the
+    predicted disparity is from its true value (equivalent to ``epe``). This is averaged over all pixels
+    of all images. ``1px``, ``3px``, ``5px`` and indicate the percentage of pixels that have a lower
+    error than that of the ground truth. ``relepe`` is the "relative-end-point-error" and is the
+    average ``epe`` divided by the average ground truth disparity. ``f1`` corresponds to the average of pixels whose epe
+    is either >3px, or whom's ``relepe`` is higher than 0.05 (therefore lower is better).
+
+    """
+
+    MEGVII = Weights(
+        # Weights ported from https://github.com/megvii-research/CREStereo
+        url="https://download.pytorch.org/models/crestereo_756c8b0f.pth",
+        transforms=StereoMatching,
+        meta={
+            **_COMMON_META,
+            "num_params": 5432948,
+            "recipe": "https://github.com/megvii-research/CREStereo",
+            "_metrics": {
+                "Middlebury2014-train": {
+                    # number of cascade stages
+                    1: {
+                        # number of refininement interations
+                        2: {
```

Comment:
This double nested key `cascade-stages` -> `refinement iterations` might break the docs and it is not clear on docs just the number `1` and `2`.

@NicolasHug any suggestions on how to make this better?
Currently I am thinking of encode the string key like: `"cascade:1-iter:2" but this looks ugly...

Reply:
I actually discussed this offline with @TeodorPoncu but forgot about the way it would be rendered in docs, and indeed it would be problematic when this is moved away from prototype:
https://github.com/pytorch/vision/blob/3c9ae0ac5dda3d881a6ce5004ce5756ae7de7bc4/docs/source/conf.py#L395

One option would be to report only on the "standard" configuration and leave the detailed breakdown as private

```
"_metrics": {
    "Middlebury2014-train": {
          # metrics for x cascades and y iterations 
          "mae": 1.704,
          "rmse": 3.738,
          "1px": 0.738,
          "3px": 0.896,
          "5px": 0.933,
          "relepe": 0.157,
          "f1": 10.041,
          "_detailed":
               # number of cascade stages
               1: {
               # number of refininement interations
                   2: {      
                    "mae": 1.704,
                    "rmse": 3.738,
                    "1px": 0.738,
                    "3px": 0.896,
                    "5px": 0.933,
                    "relepe": 0.157,
                    "f1": 10.041,
                    },
                   5: {
                   ....
                   },
              }
          }
     }   
```

Reply:
@jdsgomes yes, I agree on your suggestion. I think have the the standard one with other non-standard in private works. We just need to define what is the `metric_keys` for stereo_matching use case in the docs code later.